src/packages/cli/templates/model-migration.js

```diff
@@ -58,11 +58,11 @@ export default (name: string, attrs: Array<string> | string): string => {
         ${body}
         table.timestamps();
 
-        table.index([
+        table.index(
           'id',
```

Comment:
Indeed. That is covered. I noticed the need for separate indexes too in the comment before this. Knex has no API to add multiple indexes at once. I'll try to work on this a bit later tonight.